.travis.yml

```diff
@@ -130,7 +130,7 @@ install:
   # Install sphinx so that pylint will be able to import it
   - "[ $TESTS != lint ] || pip install sphinx==1.3.3"
   # TODO: Pin Sphinx version to workaround http://trac.buildbot.net/ticket/3408
```

Comment:
Please move this comment (`TODO: Pin Sphinx ...`) to the corresponding line in `setup.py`.